x-pack/plugin/security/cli/src/main/java/org/elasticsearch/xpack/security/cli/CertificateTool.java

```diff
@@ -247,6 +250,7 @@ final void acceptsCertificateAuthority() {
                 .withOptionalArg();
 
             acceptsCertificateAuthorityName();
+            acceptCertificateAuthorityKeyUsage();
```

Comment:
I think we want to drop this (because we don't support generating a ca on the fly any more, since 8.0 I think)

Reply:
I wasn't sure for this one and went with adding it. I've removed it. Now it's only applicable with `ca` option (https://github.com/elastic/elasticsearch/pull/126376/commits/5fdda888ae611cfd57364106e3599c655a4916dd).